### PR TITLE
Better server

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "babel-preset-react": "~6.11.1",
     "browser-sync": "~2.15.0",
     "browser-sync-webpack-plugin": "~1.1.2",
+    "bs-fullscreen-message": "~1.1.0",
```
Comment (vvo) [inline package.json:43]: We use https://github.com/glenjamin/webpack-hot-middleware, it provides a fullscreen overlay showing the error, nice

Comment (Kikobeats) [reply package.json:43]: hmm interesting, maybe is not necessary, I think that actually I'm using it: https://github.com/Kikobeats/react-boilerplatinum/blob/master/webpack.dev.config.js#L33 Can you confirm me ?

Comment (vvo) [reply package.json:43]: From reading the docs of webpack-hot-middleware, this seems different. You still need webpack-hot-middleware to be put along your entries and somewhere along with browsersync. This is the setup we have on react-instantsearch

Comment (vvo) [reply package.json:43]: Also just discovered https://github.com/facebookincubator/create-react-app/tree/master/packages/react-dev-utils, nice utils here

Comment (Kikobeats) [reply package.json:43]: maybe related? https://github.com/commissure/redbox-react

Comment (vvo) [reply package.json:43]: Yep similar also

**Pull request merged**